### PR TITLE
[Docs][SIEM] Cases UI docs

### PR DESCRIPTION
**Pull request by benskelker**
Documents Cases UI.

The relevant files are:
* https://github.com/elastic/stack-docs/pull/967/files#diff-a52c1f9d23120b52926c05690c4cadf9
* https://github.com/elastic/stack-docs/pull/967/files#diff-e2723a41f88c89f9ee5813ed450fc39d

[Preview](http://stack-docs_967.docs-preview.app.elstc.co/guide/en/siem/guide/master/cases-overview.html)

### CODE REVIEW

**Comment by cnasikas** [comment]
Hi! Great job!

Some notes:

- The `on edit and update options` determines how a field should be updated. The option can have three values: overwritten, appended (the updated value will be appended to the original value), and nothing (nothing will happen). When a comment is edited on SIEM a new comment will be created on ServiceNow (this means that the edited comment is appended). I think the phrase `When a new comment is added to a SIEM case` should change to `When a new comment is updated...` 

- [Configuring external connections](http://stack-docs_967.docs-preview.app.elstc.co/guide/en/siem/guide/master/cases-ui-integrations.html) should explain what closure options are and how you can update a connector.

File: docs/en/siem/cases-overview.asciidoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+(open or closed).
+
+To view a case, click on its name. You can then:
+
+* Add a new comment.
+* Edit existing comments and the case's description.
+* Send updates to {sn} (if external connections are configured).
+* Close the case.
+* Reopen a closed case.
+
```
Comment (stephmilovic) [inline docs/en/siem/cases-overview.asciidoc:55]: - edit tags
- refresh the case data (to get updates from other users)

File: docs/en/siem/cases-api-add-comment.asciidoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+==== Response payload
+
+A JSON object that includes the user who created the comment, and the comment's
+ID, version, and creation time.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
```
Comment (XavierM) [inline docs/en/siem/cases-api-add-comment.asciidoc:54]: related to that comment https://github.com/elastic/stack-docs/pull/761#discussion_r402361161

File: docs/en/siem/cases-api-create.asciidoc [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`title` |String |The case's title. |Yes
+|`description` |String |The case's description. |Yes
+|`status` |String a|Indicates whether the case is open or closed. Valid values
```
Comment (XavierM) [inline docs/en/siem/cases-api-create.asciidoc:20]: related to this one https://github.com/elastic/stack-docs/pull/761#discussion_r402320910

File: docs/en/siem/cases-api-create.asciidoc [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+===== Example request
+
+[source,sh]
+--------------------------------------------------
+POST api/cases
+{
+  "description": "James Bond clicked on a highly suspicious email
+  banner advertising cheap holidays for underpaid civil servants.",
+  "title": "This case will self-destruct in 5 seconds",
+  "status": "open",
```
Comment (XavierM) [inline docs/en/siem/cases-api-create.asciidoc:40]: and here

File: docs/en/siem/cases-api-find-connectors.asciidoc [hunk 5]
```diff
@@ -0,0 +1,96 @@
+[[cases-api-find-connectors]]
```
Comment (XavierM) [inline docs/en/siem/cases-api-find-connectors.asciidoc:1]: we are only getting connector as `.serviceNow` type.

**Pull request merged**